utils/config/config.go

```diff
@@ -508,7 +508,7 @@ type ServerDetails struct {
 	User                 string `json:"user,omitempty"`
 	Password             string `json:"password,omitempty"`
 	SshKeyPath           string `json:"sshKeyPath,omitempty"`
-	SshPassphrase        string `json:"SshPassphrase,omitempty"`
+	SshPassphrase        string `json:"sshPassphrase,omitempty"`
```

Comment:
Dopes this require config upgrade to v6?

Reply:
No. It's case-insensitive. See https://pkg.go.dev/encoding/json#Unmarshal:

> To unmarshal JSON into a struct, Unmarshal matches incoming object keys to the keys used by Marshal (either the struct field name or its tag), preferring an exact match but also accepting a case-insensitive match